ricochet.pro

```diff
@@ -40,6 +40,46 @@ CONFIG += c++11
 
 VERSION = 1.1.0
 
+CONFIG(hardening):unix:!macx {
```

Comment:
also, you could make this specific to the compiler, rather than the platform, using something like this to replace the "unix:!macx" construct:

*-g++* (not *-g++-*, though that would work on e.g. Linux, because you then won't match e.g. netbsd-g++)

(please test this though, that's just off the top of my head)